docs/assets/style.css

```diff
@@ -59,3 +51,7 @@
     content: "·";
     margin: 0 0.3em;
 }
+
+.carcin-play__actions > button.md-button {
+  z-index: 9;
+}
```

Comment:
Part of my comment was that with the higher numbers in other places this addition is no longer needed

```suggestion
```